### ISSUE TITLE
Bedre implementasjon av scrolling ved tastaturnavigasjon av forum, tråder og poster

### ISSUE THREAD

**Issue by havremunken** [RosenborgSupporterSoftware/RUSK#82]
Når man bruker bl.a. j/k for å navigere forum/tråder/poster vil den respektive extmod bruke scrollIntoView() for å sørge for at man faktisk ser det man er på i browser-vinduet.

Denne er ikke optimal. Den scroller når det strengt tatt ikke er nødvendig, og den scroller litt for nøyaktig noen ganger. F.eks. hvis man kommer til siste post kunne den med fordel ha scrollet ennå litt lenger ned så man så at man er på bunnen og ikke trenger å lure på om det er plass til en liten luring til lenger ned.

Så dette issuet blir et sted for å eksperimentere med bedre scrolle-stratergier.

**Comment by larsjaas** [comment]
Annen case: En tråd har plutselg i løpet av en kveld fått flere sider med nye innlegg. Mens man cacther opp, hver gang man navigerer til neste side, så flyttes man til nederste innlegget på siden, og markering/farvelegging av hva som er uleste innlegg fungerer heller ikke i den situasjonen.

Mulig RUSK burde tracke hvor langt man faktisk var kommet i tråden selv...?

**Comment by havremunken** [comment]
Ja, i det øyeblikket du går inn på en eller annen side i en tråd som har uleste meldinger, vil phpbb merke hele tråden som ulest. Det er vanskelig å omgå og eneste muligheten jeg ser er som du sier å tracke hvilken post man sist har sett i hver tråd, og bruke dette som en bedre indikasjon på hva som er ulest.

Da må man kanskje begrense seg til øverste post-id man har sett på en side, og ikke prøve å bestemme om brukeren har scrollet innom og sett posten eller ikke?

**Comment by havremunken** [comment]
Har oppdatert til å bruke scrollIntoViewIfNeeded() som oppfører seg litt bedre. Ikke perfekt, men bedre.
